src/Swarm/Game/ScenarioInfo.hs

```diff
@@ -287,7 +287,11 @@ loadScenarioDir em dir = do
   let morder' = filter (`elem` fs) <$> morder
   SC morder' . M.fromList <$> mapM (\item -> (item,) <$> loadScenarioItem em (dir </> item)) fs
  where
-  keepYamlOrDirectory = filter (\f -> takeExtensions f `elem` ["", ".yaml"])
+  -- Keep only files which are .yaml files or directories that start
+  -- with something other than a lowercase letter.
```

Comment:
Lowercase is a bit surprising, how about using some standard way to hide directories?

E.g. `.hidden`, `_private`, `internal_...` or something like that?

Reply:
Ah, good point, I think a leading underscore would work well.  I don't think we want `.hidden` since it would be annoying having your OS hide the directories if you are using them to organize `.sw` files.